src/test/scala/com/microsoft/hyperspace/index/IndexLogEntryTest.scala

```diff
@@ -377,8 +377,8 @@ class IndexLogEntryTest extends SparkFunSuite with SQLHelper with BeforeAndAfter
 
   test("Directory.fromDirectory and fromLeafFiles where there is a gap in directories.") {
     // File Structure
-    // testDir/a/f1
-    // testDir/b/c/f2
+    // testDir/temp/a/f1
+    // testDir/temp/b/c/f2
```

Comment:
This change is an resolution of [this comment](https://github.com/microsoft/hyperspace/pull/162/files#r489593365). It's repeated across many other tests.